examples/snapshot.json

```
@@ -0,0 +1 @@
+{"application":"application-sample","components":[{"name":"component-sample","containerImage":"quay.io/redhat-appstudio/sample-image@sha256:34fb1c72bc976937d0a59849160746cec12a8c925f31e74dc9f3b922f74da0e9","source":{"git":{"url":"[git@github.com](mailto:git@github.com):redhat-appstudio/integration-service.git\n","revision":"sha256:3703843e80f39188c060a6ed0f524519a682be1e7141fee29b878b1973e9783c"}}
```

I think that the formatting modified the github link, I think it should look something like this:
```suggestion
{"application":"application-sample","components":[{"name":"component-sample","containerImage":"quay.io/redhat-appstudio/sample-image@sha256:34fb1c72bc976937d0a59849160746cec12a8c925f31e74dc9f3b922f74da0e9","source":{"git":{"url":"git@github.com:redhat-appstudio/integration-service.git","revision":"sha256:3703843e80f39188c060a6ed0f524519a682be1e7141fee29b878b1973e9783c"}}
```

---

Updated. Thanks

---

Not seeing the updated changes, can you check if it looks ok on your side?